src/client/visualizations/table/table.tsx

```diff
@@ -243,34 +332,64 @@ export class Table extends BaseVisualization<TableState> {
       className: 'sort-arrow ' + commonSort.direction
     }) : null;
 
-    var cornerSortArrow: JSX.Element = null;
-    if (commonSortName === splitDimension.name) {
-      cornerSortArrow = sortArrowIcon;
-    }
+    var measureWidth = this.getIdealMeasureWidth(this.state.bodyWidth, essence);
 
-    var measuresArray = essence.getEffectiveMeasures().toArray();
+    return essence.getEffectiveMeasures().toArray().map((measure, i) => {
+      let amISorted = commonSortName === measure.name;
+      let classNames = [
+        'measure-name',
+        measure === hoverMeasure ? ' hover' : undefined,
+        amISorted ? 'sorted' : undefined
+      ].filter(Boolean);
 
-    var headerColumns = measuresArray.map((measure, i) => {
       return <div
-        className={'measure-name' + (measure === hoverMeasure ? ' hover' : '')}
+        className={classNames.join(' ')}
         key={measure.name}
+        style={{width: measureWidth}}
       >
         <div className="title-wrap">{measure.title}</div>
-        {commonSortName === measure.name ? sortArrowIcon : null}
+        {amISorted ? sortArrowIcon : null}
       </div>;
     });
+  }
+
+  renderCornerSortArrow(essence: Essence): JSX.Element {
+    var commonSort = essence.getCommonSort();
+
+    if (!commonSort) {
+      return null;
+    }
+
+    var { splits, dataSource } = essence;
+    var splitDimension = splits.get(0).getDimension(dataSource.dimensions);
+    if ((commonSort.expression as RefExpression).name === splitDimension.name) {
+      return React.createElement(SvgIcon, {
+        svg: require('../../icons/sort-arrow.svg'),
+        className: 'sort-arrow ' + commonSort.direction
+      });
```

Comment:
I know you copy pasted this but would you mind turning this into JSX syntax please